circle.yml

```diff
@@ -17,8 +17,4 @@ deployment:
   master:
     branch: master
     commands:
-      - git tag `python setup.py --version`
-      - git push --tags
-      - python setup.py bdist_wheel
-      - twine upload dist/* --username openfisca-bot --password $PYPI_PASSWORD
-      - ssh deploy-new-api@api-test.openfisca.fr
+      - sh ./deploy-if-version-bump.sh
```

Comment:
It's more usual to make the script executable with `chmod +x deploy-if-version-bump.sh`, and then just write:
```
- ./deploy-if-version-bump.sh
```

(See what's done [line 15](https://github.com/openfisca/openfisca-core/pull/547/files#diff-29944324a3cbf9f4bd0162dfe3975d88R15))